src/fusion_pipeline.py

```diff
@@ -45,15 +45,21 @@ def process_cloud(self, tilecode, points, mask=None):
         An array of shape (n_points,) with dtype=uint16 indicating the label
         for each point.
         """
-        if mask is None:
-            mask = np.full((len(points['x']),), True)
+        if mask is None:  # TODO dit wordt 2 keer uitgevoerd (ook in process_file)
+            mask = np.full(len(points['x']), True)
+
+        if self.fusers:  # TODO kan dit mooier?
+            labels = np.zeros((len(points['x']),), dtype='uint16')  # TODO vgm moet de comma hier weg
 
-        labels = np.zeros((len(points['x']),), dtype='uint16')
         for fuser in self.fusers:
             label_mask = fuser.get_label_mask(tilecode, points, mask)
             labels[label_mask] = fuser.get_label()
             mask[label_mask] = False
 
+        for grower in self.region_growing:
+            label_mask = grower.get_label_mask(points, labels, header) # TODO eigenlijk geef ik hier gewoon "las" mee
```

Comment:
argument "points" is een dict met xyz. Beter geven we gewoon de "las" mee, met header en labels

Reply:
Ipv de hele header, beter alleen dat meegeven wat echt nodig is?